app/models/user.rb

```diff
@@ -1,8 +1,8 @@
 # frozen_string_literal: true
 
 class User < ApplicationRecord
-  devise :database_authenticatable, :registerable, :trackable,
-         :rememberable, :validatable, :confirmable, :passwordless_authenticatable
+  devise :registerable, :trackable, :rememberable, :confirmable, :passwordless_authenticatable
```

Comment:
Validatable validates the format of the email so we probably need it. And it also validates password and I'm not sure it will play ball to have one and not the other.

Based on the feedback from the designer, we need to remove rememberable as well. I have altredy removed it from the UI.